README.md

```diff
@@ -70,4 +70,23 @@ A small example how this library is efficient in reading a `json` file _(more pa
 - : (Topojson.t, [ `Msg of string ]) result = Ok <abstr>
 ```
 
+### Example of building a TopoJSON object and writing it to a string
+
+```ocaml
+
+let topojs = {
+    type = "Topology";
+    objects = { example = {
```

Comment:
As it mentioned in the issue it would be better to add functions that allow you to construct a new TopoJSON value. So we end up with something like:

```ocaml
let topojson = 
  let objects = [ (* Create some objects *) ]
  let topology = Topology.v objects arcs in
  Topojson.v topology 
```